main.go

```diff
@@ -29,14 +65,14 @@ func main() {
 
 	err = config.ReadConfig(config.EnvVars.Config)
 	if err != nil {
-		log.Fatal().Err(err).Msg("failed to read github app config")
+		log.Fatal().Err(err).Msg("failed to read github app config:")
 	}
 
-	// create statsd client
-	err = datadog.CreateClient()
-	if err != nil {
-		log.Error().Err(err).Msg("failed to register dogstatsd client")
-	}
+	// // create statsd client
```

Comment:
Should we remove this, as i guess we dont use it any more ?

Reply:
removed